lms/djangoapps/course_goals/management/commands/goal_reminder_email.py

```diff
@@ -51,29 +51,43 @@ def send_ace_message(goal):
     site = Site.objects.get_current()
     message_context = get_base_template_context(site)
 
-    course_home_url = get_learning_mfe_home_url(course_key=goal.course_key)
+    course_home_url = get_learning_mfe_home_url(course_key=goal.course_key, view_name='home')
 
     goals_unsubscribe_url = settings.LEARNING_MICROFRONTEND_URL + reverse(
         'course-home:unsubscribe-from-course-goal',
         kwargs={'token': goal.unsubscribe_token}
     )
 
+    image_url = settings.STATIC_URL
+    if image_url:
+        # If the image url is a relative url prepend the LMS ROOT
+        if 'http' not in image_url:
+            image_url = settings.LMS_ROOT_URL + settings.STATIC_URL
+        image_url += 'images/'
+    language = get_user_preference(user, LANGUAGE_KEY)
+
+    # Code to allow displaying different banner images for different languages
+    # However, we'll likely want to develop a better way to do this within edx-ace
+    if image_url and language and language in ['es', 'es-419']:
+        image_url += 'spanish-'
```

Comment:
I don't have a preference either way, but if you wanted, you could indent this and remove the check about image_url existing.

Reply:
sure, updated